x-pack/plugins/ml/server/models/job_validation/job_validation.ts

```diff
@@ -77,6 +78,32 @@ export async function validateJob(
       }
 
       validationMessages = filteredBasicValidationMessages;
+      let perPartitionError = false;
+      // check if the detectors with mlcategory might have different per_partition_field values
+      // if per_partition_categorization is enabled
+      if (job.analysis_config.per_partition_categorization !== undefined) {
```

Comment:
we don't have any other validation in this file. is it possible to move this to `basicJobValidation`?

Reply:
Moved these to `basicJobValidation` here https://github.com/elastic/kibana/pull/75061/commits/bf03137577f41383eff3299fba8a7e6d9dbca214

The error messages will now look something like this

<img width="1789" alt="Screen Shot 2020-08-18 at 9 58 28 AM" src="https://user-images.githubusercontent.com/43350163/90529957-ee8fac00-e139-11ea-9cf0-7f504516cb43.png">